packages/expo-cli/src/commands/build/ios/IOSBuilder.ts

```diff
@@ -123,6 +124,15 @@ See https://docs.expo.io/versions/latest/distribution/building-standalone-apps/#
       }
       await this.produceCredentials(context, experienceName, bundleIdentifier);
     } catch (e) {
+      if (e.code === ErrorCodes.NON_INTERACTIVE) {
+        const here = terminalLink('here', 'https://expo.fyi/credentials-non-interactive');
```

Comment:
```suggestion
        const here = terminalLink('expo.fyi/credentials-non-interactive', 'https://expo.fyi/credentials-non-interactive');
```

Reply:
i know i suggested something different but i think this maybe makes it stand out a bit more!